keyboards/cannonkeys/atlas/rules.mk

```
@@ -0,0 +1,21 @@
+# MCU Name
+MCU = STM32F072xB
```

Specifying the variant (`xB`) currently prevents compiling in QMK Configurator.

```suggestion
MCU = STM32F072
```